geopandas/tests/test_plotting.py

```diff
@@ -845,17 +845,27 @@ def setup_method(self):
     def test_colors(self):
         # default uniform color
         ax = self.series.plot()
-        _check_colors(1, ax.collections[0].get_facecolors(), [MPL_DFT_COLOR])  # poly
-        _check_colors(2, ax.collections[1].get_edgecolors(), [MPL_DFT_COLOR])  # line
-        _check_colors(2, ax.collections[2].get_facecolors(), [MPL_DFT_COLOR])  # point
+        _check_colors(
+            2, ax.collections[0].get_facecolors(), [MPL_DFT_COLOR] * 2
+        )  # poly
+        _check_colors(
+            2, ax.collections[1].get_edgecolors(), [MPL_DFT_COLOR] * 2
+        )  # line
+        _check_colors(
+            2, ax.collections[2].get_facecolors(), [MPL_DFT_COLOR] * 2
```

Comment:
There's only one point in the `self.series`, should not this be `1, ax.collections[2].get_facecolors(), [MPL_DFT_COLOR] `?

Reply:
That's right, thanks for pointing that out !